stdlib/public/core/Availability.swift

```diff
@@ -45,3 +45,39 @@ public func _stdlib_isOSVersionAtLeast(
   return false._value
 #endif
 }
+
+
+/// Returns the version number for the Swift Standard Library that the code
+/// calling this was originally compiled with.
+///
+/// The version number is an arbitrary `Double` value that is monotonically
+/// increasing between toolchain releases. It does not correlate with toolchain
+/// or OS version numbers, and it is not a semantic version number.
+@_alwaysEmitIntoClient @_transparent
+public var _stdlibStaticVersion: Double {
+  return 1001.0
```

Comment:
What is the maintenance story here? When does this increment? How do we track and make sure it's always incremented?

Or is it only incremented when needed for testing? In that case it is not monotonically incrementing between toolchain releases, as a single digit can span multiple toolchain releases.

Reply:
We would increment it whenever we feel like it. Ideally we'd bump the version whenever we change stdlib behavior in an observable way. In practice, I expect we'd often leave it unchanged until we run into an issue.

It's fine to leave the version number unchanged between releases. Monotonically increasing functions are only required not to decrease.

Reply:
I updated the comments to clarify semantics a little more.